src/pages/resultsView/ResultsViewPageStoreUtils.ts

```diff
@@ -116,7 +116,10 @@ export function computeGenePanelInformation(
         const patientSequencingInfo = patientInfo[gpData.uniquePatientKey];
         const hugo = entrezToGene[gpData.entrezGeneId].hugoGeneSymbol;
 
-        if (gpData.genePanelId !== undefined) {
+        if (gpData.genePanelId && gpData.sequenced) {
```

Comment:
is there an assertion against this?

Reply:
the changes in this PR to ResultsViewPageStoreUtils.spec.ts made the relevant tests fail, and this change makes them pass again.